src/aria/widgets/form/list/List.js

```diff
@@ -41,10 +41,18 @@ module.exports = Aria.classDefinition({
         skinObj.cssClassMouseover = "x" + this._skinnableClass + "MouseOverItem_" + this._cfg.sclass;
         skinObj.cssClassFooter = "x" + this._skinnableClass + "Footer_" + this._cfg.sclass;
         var divCfg = ariaUtilsJson.copy(cfg, true, ["width", "minWidth", "maxWidth", "height", "minHeight",
-                "maxHeight", "scrollBarX", "scrollBarY"]);
+                "maxHeight", "scrollBarX", "scrollBarY", "waiAria"]);
         divCfg.sclass = skinObj.divsclass;
         divCfg.margins = "0 0 0 0";
 
+        if (cfg.waiAria) {
+            if (cfg.role !== "") {
+                this._extraAttributes += ' role="' + (cfg.role || "listbox") + '" ';
+            }
```

Comment:
Those lines can be removed, as the `role` property is not really used in this PR:
```js
            if (cfg.role !== "") {
                this._extraAttributes += ' role="' + (cfg.role || "listbox") + '" ';
            }
```